src/aks-preview/HISTORY.rst

```diff
@@ -28,6 +28,7 @@ Pending
 * Vendor new SDK and bump API version to 2023-10-02-preview.
 * Update the enum for `--os-sku` in command `az aks nodepool update` to only accept the expected Ubuntu and AzureLinux OSSKUs.
 * Update description `az aks update` and remove description about outbound ip limit.
+* Add --skip-gpu-driver-install option to node pool property in `az aks nodepool add`.
```

Comment:
Move your change history to section Pending (wait until someone else releases a new version including your changes later)  OR

pick a new version number (and update version in src/aks-preview/setup.py) to release the change once the PR got merged